scipy/fftpack/tests/test_czt.py

```diff
@@ -0,0 +1,70 @@
+# This program is public domain
```

Comment:
Just saw this: "Note that license and author info, while often included in source files, do not belong in docstrings." https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#documenting-modules